### PR TITLE
check-is-dom.js

### PR DESCRIPTION
**Pull request by AndriiChuprina**

### CODE REVIEW

File: js/scripts/object-methods/check-is-it-dom-element.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    document.getElementById('check-result').innerText = document.documentElement.innerHTML;
+}
+function checkIsDOMElement(){
+    const inputValue = document.getElementById('check-input').value;
+    const getElementById = document.getElementById(inputValue);
+    const getElementsByName = document.getElementsByName(inputValue);
+    const getElementsByClassName = document.getElementsByClassName(inputValue);
+    const getElementsByTagName = document.getElementsByTagName(inputValue);
+    let element = 0;
+    if(getElementById!==null) element++;
```
Comment (dmitry-darly-solutions) [inline js/scripts/object-methods/check-is-it-dom-element.js]: please format code

File: js/index.html [hunk 2]
```diff
@@ -101,6 +101,13 @@
             </div>
             <button class="accordion"> Check is it DOM element</button>
             <div class="panel">
+                <input type="text" id="check-input">
+                <button id="check-show" onclick="showDOM()">Show DOM</button>
+                <button id="check-is-dom" onclick="checkIsDOMElement()">Check</button>
```
Comment (dmitry-darly-solutions) [inline js/index.html]: if you don't use id in js - remove it

File: js/scripts/object-methods/check-is-it-dom-element.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    if(getElementsByName.length) element+=getElementsByName.length;
+    if(getElementsByClassName.length) element+=getElementsByClassName.length;
+    if(getElementsByTagName.length) element+=getElementsByTagName.length;
+    if(element)
```
Comment (dmitry-darly-solutions) [inline js/scripts/object-methods/check-is-it-dom-element.js:15]: I think you don't need this if

File: js/scripts/object-methods/check-is-it-dom-element.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        element>1?alert(`${inputValue} is in DOM ${element} times`):
+            alert(`${inputValue} is in DOM ${element} time`);
```
Comment (dmitry-darly-solutions) [inline js/scripts/object-methods/check-is-it-dom-element.js]: alert(`${inputValue} is in DOM ${element} ${element>1? 'times' : 'time'}`);

File: js/scripts/object-methods/check-is-it-dom-element.js (hunk 3 above)
Comment (dmitry-darly-solutions) [inline js/scripts/object-methods/check-is-it-dom-element.js]: if (!element) return;

File: js/scripts/object-methods/check-is-it-dom-element.js (hunk 4 above)
Comment (AndriiChuprina) [reply js/scripts/object-methods/check-is-it-dom-element.js]: done

File: js/index.html (hunk 2 above)
Comment (AndriiChuprina) [reply js/index.html]: done

File: js/scripts/object-methods/check-is-it-dom-element.js (hunk 3 above)
Comment (AndriiChuprina) [reply js/scripts/object-methods/check-is-it-dom-element.js]: done

File: js/scripts/object-methods/check-is-it-dom-element.js (hunk 1 above)
Comment (AndriiChuprina) [reply js/scripts/object-methods/check-is-it-dom-element.js]: done

**Review by dmitry-darly-solutions — APPROVED** [review]

**Pull request merged**